src/main/asciidoc/_chapters/ops_mgt.adoc

```diff
@@ -2521,93 +2517,36 @@ NOTE: WALs are saved when replication is enabled or disabled as long as peers ex
 [[rs.failover.details]]
 ==== Region Server Failover
 
-When no region servers are failing, keeping track of the logs in ZooKeeper adds no value.
-Unfortunately, region servers do fail, and since ZooKeeper is highly available, it is useful for managing the transfer of the queues in the event of a failure.
-
-Each of the master cluster region servers keeps a watcher on every other region server, in order to be notified when one dies (just as the master does). When a failure happens, they all race to create a znode called `lock` inside the dead region server's znode that contains its queues.
```

Comment:
I think we still need to keep this, as it is the logic for some hbase releases. Let me check the version where we start to use SCP to claim replication queue.